app/controllers/friend_lists_controller.rb

```diff
@@ -6,6 +6,17 @@ def index
     @current_user_friend_lists = FriendList.current_user_friend_lists(current_user)
 
     if @current_user_friend_lists.any?
+      @current_user_friend_lists = @current_user_friend_lists.map do |friend|
+        profile_picture_url = friend.profile_picture.attached? ? url_for(friend.profile_picture) : nil
```

Comment:
get profile_picture using .url method which is available for active storage.

Reply:
added it.